expressEngine/test/data/product/error.js

```diff
@@ -0,0 +1,15 @@
+const id = {
+  number: 123456786565,
+};
+
+const price = {
+  withString: 'sfdsafa',
```

Comment:
@ayushj117 Please use valid data instead of sfdsafa

Reply:
Done sir